src/cli/backup/groups.go

```diff
@@ -36,26 +36,25 @@ const (
 	groupsServiceCommandDetailsUseSuffix = "--backup <backupId>"
 )
 
-// TODO: correct examples
 const (
-	groupsServiceCommandCreateExamples = `# Backup all Groups data for Alice
-corso backup create groups --group alice@example.com 
+	groupsServiceCommandCreateExamples = `# Backup all Groups and Teams data for the Marketing group
+corso backup create groups --group Marketing
 
-# Backup only Groups contacts for Alice and Bob
-corso backup create groups --group engineering,sales --data contacts
+# Backup only Teams conversations messages
+corso backup create groups --group Marketing --data messages
```

Comment:
Are `names` supported for groups? I am able to work only by providing `ID` with current main.

Reply:
They are not supported at this time.  I'll be investigating this shortly.

Reply:
@neha-Gupta1 
that support has been added.  See: https://github.com/alcionai/corso/pull/4242

Reply:
Perfect